Updated readme on using/testing

abbyli437 opened the pull request:
Updated readme to link info on documentation for how to use and test our feature

abbyli437 committed:
Updated Readme on Using and Testing info

abbyli437 committed:
Fixed typo from previous commit

Fixed typo that didn't properly display hyperlink for "Using and Testing" in previous commit

abbyli437 committed:
Changed spacing and line breaks of hyperlink

changed spacing and line breaks of hyperlink for "using and testing" to see if it will appear

victoralfonzo reviewed (approved):
Looks good!

coffeeologist commented:
Actually, thoughts on moving our stuff right under all the tags (before the "Mayan" picture), so that it's slightly easier to find / more noticeable?

abbyli437 committed:
moved update/testing section to top

moved update/testing section of readme to top of the document based on Amy's suggestion

abbyli437 committed:
added line between update/testing and rest of doc

added line between Update/Testing section of readme and rest of the doc for clarity

abbyli437 committed:
fixed dividing line placement in readme

previous commit put the line between Update/Testing and rest of doc a line too early, shifted line to correct position

coffeeologist commented:
Thanks for the change Abby!
